### PR TITLE
Move the main contracts from solidity/contracts to contracts and remove unneeded scripts

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

**Review by yudilevi — APPROVED** [review]

**Review by sshmaxime — APPROVED** [review]

File: hardhat.config.js [hunk 1]
```diff
@@ -62,21 +62,21 @@ module.exports = {
...
+        tests: './test',
         cache: './cache',
-        artifacts: './solidity/build/contracts'
+        artifacts: './build/contracts'
     },
 
     // Test Config
     mocha: {
-        spec: argv.spec || 'solidity/test',
+        spec: argv.spec || 'test',
```
Comment (barakman) [inline hardhat.config.js:79]: IMO, this is a good opportunity to rename folder `test` (Truffle's default) to `tests`.

File: hardhat.config.js [hunk 2]
```diff
@@ -62,21 +62,21 @@ module.exports = {
...
-        path: './solidity/build/abi',
+        path: './build/abi',
         clear: true
     },
 
     // System Config
     paths: {
-        sources: './solidity/contracts',
-        tests: './solidity/test',
+        sources: './contracts',
```
Comment (barakman) [inline hardhat.config.js:71]: IMO, this is a good opportunity to rename folder `test` (Truffle's default) to `tests`.

**Review by barakman — CHANGES_REQUESTED** [review]

File: hardhat.config.js (hunk 1 above)
Comment (yudilevi) [reply hardhat.config.js:79]: we should just use the standard terminology in hardhat.

Comment (lbeder) [reply hardhat.config.js:79]: 👍. I'm moving it to the defaults (`test` remains, and `build` is changed to `artifacts`)

**Comment by lbeder** [comment]
Please note that I've reverted to use Hardhat's defaults (while moving the abi-exporter output to `data/abi`, as it can't be at the same default folder).

**Review by yudilevi — APPROVED** [review]

**Review by barakman — APPROVED** [review]

**Review by sshmaxime — APPROVED** [review]

**Pull request merged**